base_app/src/app/app.component.ts

```diff
@@ -1,18 +1,18 @@
 import {AfterViewInit, Component, EventEmitter, OnInit, Output, ViewChild} from '@angular/core';
-import './css/styles.css';
+// import './css/styles.css';
 import {AppProperties, SubscriptionState, TabUI, TitleBarProperties} from './interfaces/props-data';
 import {ajax} from 'rxjs/ajax';
 import {MatTabChangeEvent, MatTabGroup} from '@angular/material/tabs';
 import {interval} from 'rxjs';
 import {ClientLogger, LogLevel} from '../tools/logger';
 import {DataSummary} from './interfaces/data-summary';
 import {UTIL} from '../tools/utility';
-import {AppEditUiPanelComponent} from './app-tab-overlay/app-edit-ui-panel-component';
-
+// import {AppEditUiPanelComponent} from './app-tab-overlay/app-edit-ui-panel-component';
```

Comment:
commented lines?

Reply:
Removed along with some other comments.
Left some comments that have some functionality that we still might want to restore later.